remote-api/src/main/java/org/alfresco/rest/framework/core/ResourceInspector.java

```diff
@@ -64,6 +64,7 @@
 import org.alfresco.rest.framework.resource.actions.interfaces.RelationshipResourceAction;
 import org.alfresco.rest.framework.resource.actions.interfaces.RelationshipResourceBinaryAction;
 import org.alfresco.rest.framework.resource.actions.interfaces.ResourceAction;
+import org.alfresco.rest.framework.resource.actions.interfaces.FolderResourceAction;
```

Comment:
All the changes in this class are unnecessary and should be removed, as they have no impact on the actual feature. It's important to keep the code clean and focused on the functionality that directly affects the feature.